crates/sui-protocol-config/src/lib.rs

```diff
@@ -109,13 +109,21 @@ impl SupportedProtocolVersions {
 pub struct Error(pub String);
 
 /// Records on/off feature flags that may vary at each protocol version.
-#[derive(Default, Clone, Serialize)]
+#[derive(Clone, Serialize)]
 struct FeatureFlags {
     // Add feature flags here, e.g.:
     // new_protocol_feature: bool,
     package_upgrades: bool,
 }
 
+impl Default for FeatureFlags {
+    fn default() -> Self {
+        Self {
+            package_upgrades: true,
```

Comment:
It looks like you got rid of the test above that checks that upgrades are gated off -- if we're doing that, we should just get rid of the flag altogether, but we should wait on that for the adapter support.

Reply:
Fair point, I was wondering if we should still keep these flagged off or if this PR should "unflag it" but unflagging once the loader/adapater support lands seems like a better time to do that. I'll re-enable this flag and we can remove later on :)